rasa_core/utils.py

```diff
@@ -319,6 +323,21 @@ def read_yaml_file(filename):
 
         return yaml_parser.load(read_file(filename))
 
+def read_yaml_string(string):
```

Comment:
can we then rewrite `read_yaml_file` to just be `read_yaml_string(read_file(filename))` ?